recipe/meta.yaml

```diff
@@ -27,8 +27,9 @@ requirements:
     - jupyter_server >=2.0.0,<3.0.0
     - jupyter_ydoc >=1.0.1,<2.0.0
     - ypy-websocket >=0.12.1,<0.13.0
-    - jupyter_events
+    - jupyter_events <0.7.0
     - jupyter_server_fileid >=0.6.0,<1
+    - jsonschema[format-nongpl,format_nongpl] <4.18.0
```

Comment:
yeah, that syntax only works in `pip`.

Here, i'd go with:

```
- jsonschema <4.18.0
- jsonschema-with-format-nongpl
```

until  https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/496 is in